File: BackEnd/Affection-App/Common/Authentication/AuthResponse.cs


namespace Affection.Contract.Authentication;
public record AuthResponse
(
    string Id ,
    string? UserName ,
    string? Email ,
    string KnowAs ,
    string Country ,
    string City ,
    string Gender ,
    DateTime DateOfBirth ,
    string Token,
    int ExpiresIn,
    string RefreshToken,
    DateTime RefreshTokenExpiration,
    string? PhotoUrl
    
);

File: FrontEnd/Affection-App/affection-app/src/app/core/models/authentication/authResponse.ts

export interface authResponse {
   
    id: string;
    userName?: string;
    email?: string;
    knowAs: string;
    country: string;
    city: string;
    gender: string;
    dateOfBirth: Date;
    token: string;
    expiresIn: number;
    refreshToken: string;
    refreshTokenExpiration: Date;
    photoUrl?: string;
    emailConfirmed?: boolean; 
}